beetsplug/lyrics.py

```diff
@@ -395,13 +395,15 @@ def fetch(self, artist, title):
 
         song_info = None
         for hit in json["response"]["hits"]:
-            if hit["result"]["primary_artist"]["name"] == artist:
+            if hit["result"]["primary_artist"]["name"].strip(u'\u200b').lower() == artist.lower():
```

Comment:
Can you please put the left-hand side into a variable, like this, so the line doesn't get to long?

```py
hit_artist = hit["result"]["primary_artist"]["name"].strip(u'\u200b').lower()
```

Also, it might be really useful to have a comment here that describes why the odd-looking `.strip(u'\u200b')` is necessary.

Reply:
Sounds good, I wasn't clear on when comments are used in the codebase and whether long lines were an issue for now, but I agree! Will address your comments 😄